### PR TITLE
chore: Centralize & Improve Storybook configuration

### PR DESCRIPTION
**Pull request by lukelooker**

### CODE REVIEW

File: storybook/package.json [hunk 1]
```diff
@@ -6,7 +6,19 @@
...
-    "build": "build-storybook -c .storybook"
+    "build": "build-storybook -c .storybook",
+
+    "storyshots-build": "export storybookBuildMode=fast && yarn build",
+    "storyshots-preview-backup": "mv .storybook/preview.js .storybook/preview.js.bu",
+    "storyshots-preview-restore": "mv .storybook/preview.js.bu .storybook/preview.js",
+
+    "prestoryshots-task": "yarn storyshots-build && yarn storyshots-preview-backup",
+    "storyshots-task": "yarn jest",
+    "posttoryshots-task": "yarn storyshots-preview-restore",
```
Comment (ElliotTheHuman) [inline storybook/package.json]:
```suggestion
    "poststoryshots-task": "yarn storyshots-preview-restore",
```

File: storybook/.storybook/componentsDecorator.tsx [hunk 2]
```diff
@@ -28,5 +28,5 @@ import React from 'react'
 import { ComponentsProvider } from '@looker/components-providers'
 
 export const componentsDecorator = (storyFn: any) => (
-  <ComponentsProvider loadGoogleFonts>{storyFn()}</ComponentsProvider>
```
Comment (lukelooker) [inline storybook/.storybook/componentsDecorator.tsx:31]: We already do this via the `preview-head.html` file

File: storybook/.storybook/manager-head.html [hunk 3]
```diff
@@ -1,6 +0,0 @@
-<style>
-  /* Hide Focused stories from the sidebar – they're for screenshots only */
-  [id$='-focused'] {
-    display: none !important; /* stylelint-disable-line declaration-no-important */
-  }
-</style>
```
Comment (lukelooker) [inline storybook/.storybook/manager-head.html:6]: Replaced this with:

```docs: { disable: true },```

In the individual stories were stories shouldn't be published.

File: storybook/.storybook/preview.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ import {
   Primary,
   Stories,
   PRIMARY_STORY,
-} from '@storybook/addon-docs/dist/cjs/blocks'
+} from '@storybook/addon-docs/blocks'
```
Comment (lukelooker) [inline storybook/.storybook/preview.js:10]: THE ACTUAL PROBLEM.

**Review by ElliotTheHuman — APPROVED** [review]
👍 👍 👍

**Pull request merged**